tests/system-level/anemoi_test/nodes.py

```diff
@@ -0,0 +1,124 @@
+import os
+from os import path
+
+import pyflow as pf
+import wellies as wl
+import yaml
+
+SUITE_DIR = path.join(path.dirname(path.realpath(__file__)))
+
+
+def parse_training_directory(training_directory: str) -> tuple[dict]:
+    task_file = path.join(training_directory, "task_config.yaml")
```

Comment:
Maybe do the same check as for the datasets, just for symmetry? And maybe update the error to print something more meaningful?
```
            if not path.exists(task_file):
                raise FileNotFoundError(f"Training test requires a config file: {task_file}")

```